File: js/pushPop.js
let arr =[[1,2,3],4,5,6,7,8,9,10]

console.log('====================================');
console.log('before pop',arr);
console.log('====================================');
arr.push(11);
console.log('after push',arr);
console.log('====================================');
arr.at(0).push(13)
console.log('after push at index 0',arr);




console.log('====================================');
console.log('before pop',arr);
console.log('====================================');
arr.pop()
console.log('after pop',arr);
console.log('====================================');
arr.at(0).pop()
console.log('after pop at index 0',arr);

File: go/pushpop.go
package main

import "fmt"

//push

func main() {
	arr := [] int { 1, 2, 3, 4}
	fmt.Println("before push",arr)
	//push
	arr = append(arr, 5)
    fmt.Println("after push",arr)

	//pop
	popped := arr[len(arr)-1]
    arr = arr[:len(arr)-1]
    fmt.Println("after pop",arr)
    fmt.Println("popped element:", popped)
}